index.html

```diff
@@ -0,0 +1,176 @@
+        </h1>
+        <div class="ideal">
+          <span>
+            Learning is the process of acquiring knowledge <strong>skills</strong>  and <strong>values</strong>. Learning needs to be planned and evaluated. It can be measured, and its effectiveness can be assessed. <br>
+            A quality system needs to be in place that monitors and evaluates the learning process. The same goes for eLearning.
+          </span>
+          <h3 class="work-hour">8am - 5pm(MON) ~ (FRI) 8pm</h3>
+          “E-Learning doesn't just "happen"! ...
+        </div>
+      </section>
+      <section class="program" id="program">
+        <h3 class="program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/forum.png" alt="forum-icone" />
+              <span>Forum</span>
+            </div>
+            <div class="right">
+              Have the time to share your toughts and opinions with our team for
+              each topic.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/lecture.png" alt="lecture-icon" />
+              <span>Donation</span>
+            </div>
+            <div class="right">
+              The value of life is not in its duration, but in its donation.
+              What a privilege to be here on the planet to contribute your
+              unique donation to humankind!
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/support.png" alt="support-icone" />
+              <span>Support</span>
+            </div>
+            <div class="right">
+              Whatever little it maybe, it means a lot to someone out there.
+              Support the needy in our community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/workShop.png" alt="workshop-icone" />
+              <span>Workshop</span>
+            </div>
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="sponsor" class="sponsor"></section>
```

Comment:
- [ ] Kindly try to match the Behance design in the `featured speakers` section on the home page mobile version by increasing the `image size` of the speaker and making the text start from the beginning.

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164564604-5c2cb312-f324-4bf9-82b6-efc528d7d473.png) | ![image](https://user-images.githubusercontent.com/24703171/164564621-7b40365a-cb19-4769-b158-7bbbdfe01e77.png) |

- [ ] and also the same in desktop version

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164564884-6a805b90-cb41-41d7-acd3-dbb21e81c756.png) | ![image](https://user-images.githubusercontent.com/24703171/164564787-cb5dc47e-84c6-44db-9d32-bf9929f05002.png) |